code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -192,6 +203,128 @@
 	_color = "red"
 	update_icon()
 
+/obj/item/weapon/melee/energy/sword/pirate/can_combine_with(obj/item/W)
+	return 0
+
+/*
+ * Double-Bladed Energy Swords
+ */
+
+/obj/item/weapon/melee/energy/sword/dualsaber
+	icon_state = "dualsaber0"
+	name = "double-bladed energy sword"
+	desc = "Handle with care."
+	var/colorset = ""
+	force = 3
+	throwforce = 5.0
```

Comment:
uh, why 5.0?  The .0 specifically.  I don't remember ever seeing a throwforce as a float and also why if it's going to be 0 anyway